L125: python-absl-logging-support.md

```diff
@@ -0,0 +1,91 @@
+Title
+----
+* Author(s): sreenithi
+* Approver: sergiitk, asheshvidyut
+* Status: Draft
+* Implemented in: Python
+* Last updated: 04-07-2025
+* Discussion at: <google group thread> (filled after thread exists)
+xxxxxxx
+## Abstract
+
+This proposal describes the changes necessary to add absl logging support in gRPC Python hence solving the user warning:
+
+```sh
+WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
+``` 
+
+
+## Background
+
+gRPC Core changed its logging system to use absl logging as per [L117](https://github.com/grpc/proposal/blob/master/L117-core-replace-gpr-logging-with-abseil-logging.md).
+
+After this change, gRPC python users started seeing the warning log - 
+`WARNING: All log messages before absl::InitializeLog() is called are written to STDERR`.
+
+For gRPC C++, the users are expected to invoke `absl::InitializeLog()`, but gRPC Python users cannot invoke a C++ API call. Hence this proposal explains the changes necessary to gRPC Python to implicitly invoke `absl::InitializeLog()` in the Cython layer at the time of initialization.
+
+> Note that if abseil fixes https://github.com/abseil/abseil-cpp/issues/1656, gRPC Core itself will be able to invoke absl::InitializeLog(), but until then this is a workaround.
+
+### Related Proposals: 
+* https://github.com/grpc/proposal/blob/master/L117-core-replace-gpr-logging-with-abseil-logging.md
```

Comment:
nit: we tend to use [reference links](https://github.github.com/gfm/#reference-link) for this. Example: https://github.com/grpc/proposal/blob/343b4d8a21f4724d75ced970cc1fc00c997ef4d5/A83-xds-gcp-authn-filter.md?plain=1#L26-L35